node/e2etests/e2etests.js

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+var serviceSDK = require('azure-iothub');
+var deviceSDK = require('azure-iot-device');
+var deviceAMQP = require('azure-iot-device-amqp');
+var deviceHTTP = require('azure-iot-device-http');
+
+var ConnectionString = require('azure-iot-common').ConnectionString;
+
+var assert = require('chai').assert;
+var debug = require('debug')('e2etests');
+var uuid = require('uuid');
+
+var runTests = function (DeviceTransport, hubConnStr, deviceConStr, deviceName) {
+    describe('Device connected over ' + DeviceTransport.name + ':', function () {
```

Comment:
I thought about it but didn't know we were going to do it. then we need to add module.exports to the iothub explorer code I think